conf/messages.en

```diff
@@ -112,7 +113,7 @@ liabilityStartDatePage.title = What is the date you first became liable?
 liabilityStartDatePage.hint = For example, 12 11 2007
 liabilityStartDatePage.question = What is the date you first became liable?
 liabilityStartDate.formatting.error = Liability start date - wrong format
-liabilityStartDate.outOfRange.error = Liability start date must be on or after 1 April 2022 and before 2 April 2023
+liabilityStartDate.outOfRange.error = Liability date must be today or in the past
```

Comment:
I know we discussed this in 3As but in retrospect should this be more along the lines of `Liability start date must be on or after 1 April 2022`? Stating `in the past` specifically could lead to a lot of "but 31/03/2022 is in the past?" type queries... I'm not sure, what do you think?

Reply:
The error message in this context is to be decided by the content people.